### PR TITLE
expose the webpack inject option

### PR DESCRIPTION
**Pull request by the-destro**
Expose the webpack inject option '(as a boolean)' to allow more controol over the injection of scripts. Would help address #3323

### CODE REVIEW

**Comment by the-destro** [comment]
I just signed the CLA.

**Comment by clydin** [comment]
Templating doesn't work very well right now nor is it really a supported scenario.  So this wouldn't really do much.

**Comment by the-destro** [comment]
true it is not very robust but allowing some small flexibility to where the scripts are injected would be rather useful. This code would allow something as simple as including this block where needed on the page:```<% for (var chunk in htmlWebpackPlugin.files.chunks) { %>
  <script src="<%= htmlWebpackPlugin.files.chunks[chunk].entry %>"></script>
  <% } %>```

File: packages/angular-cli/models/webpack-build-common.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export function getWebpackCommonConfig(
   appConfig: any,
   baseHref: string,
   sourcemap: boolean,
-  vendorChunk: boolean
+  vendorChunk: boolean,
```
Comment (grizzm0) [inline packages/angular-cli/models/webpack-build-common.ts]: Remove trailing comma here and throughout

**Comment by the-destro** [comment]
Closing as it seems there is not upstream interest for now. Plus it is easy enough add back in

**Pull request closed without merging**